specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2023-03-15-preview/examples/GetHybridIdentityMetadata.json

```diff
@@ -0,0 +1,27 @@
+{
+  "parameters": {
+    "api-version": "2023-03-15-preview",
+    "subscriptionId": "fd3c3665-1729-4b7b-9a38-238e83b0f98b",
+    "resourceGroupName": "testrg",
+    "machineName": "ContosoVm",
+    "metadataName": "default"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/fd3c3665-1729-4b7b-9a38-238e83b0f98b/resourceGroups/testrg/providers/Microsoft.HybridCompute/machines/ContosoVm/hybridIdentityMetadata/default",
+        "name": "testItem",
+        "type": "Microsoft.HybridCompute/machines/hybridIdentityMetadata",
+        "properties": {
+          "vmId": "f8b82dff-38ef-4220-99ef-d3a3f86ddc6c",
```

Comment:
>vmId [](http://example.com/codeflow?start=11&length=4)

What does this vmid refer to ? please rename to something that does not end in id since it is not the resource id of the VM

Reply:
this is the same vmId that appears under properties in the parent /machines resource. If we change the name here, particularly if we dropped the -id, it would lose that connection and cause confusion.

Reply:
@raosuhas